x/ccv/provider/keeper/proposal.go

```diff
@@ -123,9 +126,13 @@ func (k Keeper) SetConsumerClient(ctx sdk.Context, chainID, clientID string) {
 }
 
 // GetConsumerClient returns the clientID for the given chainID.
-func (k Keeper) GetConsumerClient(ctx sdk.Context, chainID string) string {
+func (k Keeper) GetConsumerClient(ctx sdk.Context, chainID string) (string, bool) {
```

Comment:
Can we change these functions to `SetConsumerClientId` and `GetConsumerClientId`? Was confused why the return value was string for a second